config/forms/2021_2022.json

```diff
@@ -4425,10 +4425,10 @@
                       "value": "Tenant applied directly (no referral or nomination)"
                     },
                     "3": {
-                      "value": "Private registered provider (PRP) lettings only – nominated by a local housing authority"
+                      "value": "Nominated by a local housing authority"
                     },
                     "4": {
-                      "value": "Private registered provider (PRP) supported lettings only – referred by local authority housing department"
+                      "value": "Referred by local authority housing department"
```

Comment:
@roobottom Do we need to include the text ‘Private registered provider (PRP) [supported] lettings only’ given these options are only shown in this case anyway?